metering/configuring-metering/metering-about-configuring.adoc

```diff
@@ -0,0 +1,16 @@
+[id="metering-about-configuring"]
+= About configuring metering
+include::modules/common-attributes.adoc[]
+:context: metering 
+
+toc::[]
+
+A `CustomResourceDefinition` (CRD) called your `MeteringConfig` specifies all the configuration details for your metering installation. When you first install the metering stack, a default `MeteringConfig` is generated. Use the examples in the documentaion to modify this default file. Keep in mind the following key points:
```

Comment:
Does the user edit the CRD or the CR? In Cluster Logging and Node, the user doesn't touch the CRD but makes all changes in the CR.

Reply:
Confirmed that it is the CR. I will update.